data/united-states/tennessee.csv

```diff
@@ -8,7 +8,6 @@ a85f70a5-1b54-43ba-8ab3-261e20f8d570,Balter Beerworks,brewpub,100 S Broadway St,
 8f3af559-423d-42c8-903e-fe45750307cb,Beale St Brewing Co,planning,,,,Memphis,Tennessee,38104-6826,United States,9014917197,http://www.bealestbrewing.com,,
 667547fa-dcf1-4b05-9f59-53c6c7a24847,Bearded Iris Brewing,micro,101 Van Buren St,,,Nashville,Tennessee,37208-1719,United States,6159287988,http://www.beardedirisbrewing.com,-86.786174,36.181424
 4e558cde-490f-4c49-b0c0-28d3abb9458b,Big Frog Brewing Company,micro,2122 Dayton Blvd,,,Red Bank,Tennessee,37415-6414,United States,4238039046,http://www.bigfrogbrewery.com,-85.31017603,35.0916037
-002012b4-afef-4e35-95bf-56930dbc599e,Big River Grille & Brewing Works - Chattanooga,brewpub,222 Broad St,,,Chattanooga,Tennessee,37402-1009,United States,4232672739,http://www.craftworksrestaurants.com,-85.31070875,35.05387554

```

Comment:
Instead of removing, let's mark this and the others as "closed" type. We're working on building a history table, so this will be useful to track.